.gitignore

```diff
@@ -4,5 +4,3 @@
 .*.swp
 /.vscode
 cover.out
-vendor
-/.idea
```

Comment:
I think this part should not be reverted.

Reply:
That's a problem with my local environment, so it's ok.